bin/eslint.js

```diff
@@ -16,6 +16,8 @@ var cli = new CLIEngine(options);
 var debug = false;
 var checks = require("../lib/checks");
 
+console.error("ESLint is running with the " + cli.getConfigForFile(null).parser + " parser.");
```

Comment:
Since we already set `console.log` to `console.error` to achieve the right semantics re output streams automatically, I'd probably use `console.log` for our own logging too, to be consistent. WDYT?